### ISSUE TITLE
Alt-Arrow key is quantized but not stepped

### ISSUE THREAD

**Issue by baconpaul** [surge-synthesizer/surge#6593]
Chessel on discord says:

> No.  To go up one semitone I have to press up 8 times with control held down before it registers going up a semitone.  I just noted that alt + up arrow also takes 8 up arrow presses but settles on the intermediate values as it goes up whereas control ++ up arrow stays quantized on whole values.
> There is an additional querk where if you use control up arrow to go up a semitone, a single control down arrow will jump the whole semitone in one press.  but further control arrow down returns to the 8 keystrokes required behaviour.

What is happening is the alt is quantizing the move but not changing the amount the move is applied. So you press up and you get jogged the same amount as if alt isn't down, but that doesn't update until you jog 'enough'. Sigh.

May be tricky to fix since that quantization jog distance isn't advertised by a param. But toss it into 112 anyway.

**Comment by baconpaul** [comment]
sigh this is really hard and we should do it differently in XT2 and SC

Basically the parameter knows its step size and the slider is 0...1 which is fine-ish but in this case it means we don't know how far to jog and don't have an API to jog up by one. The quantization happens in `Parameter::set_value_f01` which calls `Parameter:;bound_value`

Now, `Parameter::set_value_f01` doesn't require an instance really so I could set the control type onto the slider and fake up the search for the 'next' step up or down. Let me look at that. But if that hack doesn't work in the keybinding, then we are out of luck for XT1.*
